humans/hair_styles_and_colors.md

```diff
@@ -4,31 +4,80 @@ Valid hair styles:
 
 ```json
 [
-  "female_hair_bob_1",
-  "female_hair_long_curly"
-  "female_hair_long_curly_1",
-  "female_hair_long_simple_1",
-  "female_hair_long_wavy_1",
-  "female_hair_long_generic_1",
-  "female_hair_long_layered_1",
-  "female_hair_mid_messy_1",
-  "female_hair_long_chandelierlayers",
-  "female_hair_short_90s",
-  "female_hair_medium_bob",
-  "male_hair_combover_1",
-  "male_hair_mohawk_1",
-  "male_hair_short_curly_1",
-  "male_hair_fauxhawk_1",
-  "male_hair_long_messy_1",
-  "male_hair_short_messy_1",
-  "male_hair_short_sweptback_1",
-  "male_hair_quiff_1",
-  "male_long_straight",
-  "male_hair_short_fadepart",
-  "male_hair_short_frenchcrop"
+  "bob_01",
+  "bob_02",
+  "buzzcut_01",
+  "buzzcut_02",
+  "convertible_01",
+  "curtain_01",
+  "curtain_02",
+  "curtain_03",
+  "curtain_04",
+  "curtain_05",
+  "curtain_06",
+  "curtain_07",
+  "curtain_08",
+  "curtain_09",
+  "curtain_10",
+  "frenchcrop_01",
+  "frenchcrop_02",
+  "frenchcrop_03",
+  "layered_01",
+  "layered_02",
+  "layered_03",
+  "layered_04",
+  "layered_05",
+  "layered_06",
+  "messy_01",
+  "mohawk_01",
+  "mohawk_02",
+  "pompadour_01",
+  "pompadour_02",
+  "quiff_01",
+  "quiff_02",
+  "simple_01",
+  "simple_02",
+  "slicked_01"
 ]
 ```
+Mapping of old hair styles to new hair styles:
 
+|           Old hair style                 |   New hair style  |
+|------------------------------------------|-------------------|
+| "female_hair_bob_1"                      | "bob_01"          |
+| "female_hair_long_chandelierlayers"      | "layered_01"      |
+| "female_hair_long_curly"                 | "layered_02"      |
+| "female_hair_long_curly_1"               | "layered_03"      |
+| "female_hair_long_generic_1"             | "simple_01"       |
+| "female_hair_long_layered_1"             | "layered_04"      |
+| "female_hair_long_simple_1"              | "simple_02"       |
+| "female_hair_long_wavy_1"                | "layered_05"      |
+| "female_hair_medium_bob"                 | "bob_02"          |
+| "female_hair_mid_messy_1"                | "messy_01"        |
+| "female_hair_short_90s"                  | "convertible_01"  |
+| "male_hair_combover_1"                   | "quiff_01"        |
+| "male_hair_curtain_long"                 | "curtain_01"      |
+| "male_hair_curtain_long_cover_eye_both"  | "curtain_02"      |
+| "male_hair_curtain_long_cover_eye_left"  | "curtain_03"      |
+| "male_hair_curtain_long_cover_eye_right" | "curtain_04"      |
+| "male_hair_curtain_long_ear_tuck_both"   | "curtain_05"      |
+| "male_hair_curtain_long_ear_tuck_left"   | "curtain_06"      |
+| "male_hair_curtain_long_ear_tuck_right"  | "curtain_07"      |
+| "male_hair_curtain_long_no_part"         | "slicked_01"      |
+| "male_hair_curtain_long_part_left"       | "curtain_08"      |
+| "male_hair_curtain_long_part_right"      | "curtain_09"      |
+| "male_hair_curtain_long_short"           | "curtain_10"      |
```

Comment:
we can delete this lines, as they are new styles that weren't ever published under their original names